exonum/src/api/public/blockchain_explorer.rs

```diff
@@ -141,9 +142,13 @@ impl ExplorerApi {
             "Expected tx in database",
         );
 
-        Ok(self.blockchain.tx_from_raw(raw_tx.clone()).ok_or_else(|| {
-            ApiError::InternalError(format!("Service not found for tx: {:?}", raw_tx).into())
-        })?)
+        Ok(self.blockchain.tx_from_raw(raw_tx.clone()).or_else(
```

Comment:
Side note: I find it mildly worrying that there are multiple places with `tx_from_raw(raw_tx.clone())` pattern. Maybe, it would make more sense to pass a raw message by reference?

Also, imo this would be more readable without `Ok(..)` wrapping:

```rust
self.blockchain
    .tx_from_raw(raw_tx.clone())
    .map_err(|e| ...)
```